# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/stefanp/psi/hotfixes.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./src/ui_about.ui.orig	2005-10-31 18:07:00.000000000 +0100
+++ ./src/ui_about.ui	2005-10-31 18:07:06.000000000 +0100
@@ -334,7 +334,6 @@
 <layoutdefaults spacing="6" margin="11"/>
 <includehints>
     <includehint>fancylabel.h</includehint>
-    <includehint>urllabel.h</includehint>
     <includehint>psitextview.h</includehint>
     <includehint>psitextview.h</includehint>
     <includehint>psitextview.h</includehint>
--- ./src/options/opt_appearance-ui.ui.orig	2005-10-31 18:11:43.000000000 +0100
+++ ./src/options/opt_appearance-ui.ui	2005-10-31 18:11:53.000000000 +0100
@@ -458,9 +458,5 @@
 </tabstops>
 <layoutdefaults spacing="6" margin="11"/>
 <includehints>
-    <includehint>fontlabel.h</includehint>
-    <includehint>fontlabel.h</includehint>
-    <includehint>fontlabel.h</includehint>
-    <includehint>fontlabel.h</includehint>
 </includehints>
 </UI>
